networklibrary/src/main/java/com/eis/communication/network/commands/AcceptInvite.java

```diff
@@ -0,0 +1,29 @@
+package com.eis.communication.network.commands;
+
+import com.eis.communication.Peer;
+import com.eis.communication.network.NetSubscriberList;
+
+/**
+ * Command to accept an Invitation to the network
+ *
+ * @author Marco Cognolato
+ * @author Giovanni Velludo
+ */
+public abstract class AcceptInvite<T extends Peer> extends Command {
```

Comment:
Wouldn't it be clearer to have `AcceptInvite<I extends Invitation>`? This way it would be clearer that in order to accept an invitation you need to have it in the first place, not just who sent it.

Reply:
🤔 Yeah that makes sense 👍